pyasic/miners/backends/luxminer.py

```diff
@@ -151,7 +151,7 @@ async def upgrade_firmware(self) -> bool:
             bool: True if the firmware upgrade was successfully initiated, False otherwise.
         """
         try:
-            await self.rpc.upgraderun()
+            await self.rpc.updaterun()
```

Comment:
Revised method name since `updaterun` is correct name in lux api
See [comment](https://github.com/UpstreamData/pyasic/pull/276/files#r1919970461)